cms/templates/cms/benefits_page.html

```diff
@@ -49,6 +49,17 @@ <h1 class="title">
         {% endblock %}
         </div>
       </div>
+      {% if '/organizations/' in request.path %}
+        <div class="mdl-cell mdl-cell--4-col sticky-top">
+          <script charset="utf-8" type="text/javascript" src="//js.hsforms.net/forms/shell.js"></script>
+          <script>
+            hbspt.forms.create({
+              portalId: "{{ hubspot_portal_id | safe }}",
+              formId: "{{ hubspot_ogranizations_form_guid | safe }}"
+            });
+          </script>
```

Comment:
It will help in performance optimization.
```suggestion
          <script defer charset="utf-8" type="text/javascript" src="//js.hsforms.net/forms/shell.js"></script>
          <script>
          document.addEventListener("DOMContentLoaded", function() {
                hbspt.forms.create({
                  portalId: "{{ hubspot_portal_id | safe }}",
                  formId: "{{ hubspot_ogranizations_form_guid | safe }}"
                });
            }
          </script>
```

Reply:
It will be handled in a future PR, so resolving this comment.